### PR TITLE
Add jquery.turbolinks@2.1.0 /w git auto-update & Update package.json

### LINKED ISSUE #7254: Update jquery.turbolinks
https://cdnjs.com/libraries/jquery.turbolinks
https://github.com/kossnocorp/jquery.turbolinks

### PR DESCRIPTION
**Pull request by sufuf3**
close #7604, close #7254
Pull request for issue: #7604
Related issue(s): #7254 #11170 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/kossnocorp/jquery.turbolinks
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/jquery.turbolinks
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 25
   - Count of stars: 1049
   - Count of forks: 60
 * NPM download stats (optional):
   - Downloads in the last day: 23
   - Downloads in the last week: 245
   - Downloads in the last month: 1,061 

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [X] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
@cdnjs/intern3 @dakshshah96 
Please help me review this PR, Thank you.

File: ajax/libs/jquery.turbolinks/package.json [hunk 1]
```diff
@@ -1,11 +1,23 @@
 {
   "name": "jquery.turbolinks",
-  "version": "2.0.2",
+  "version": "2.1.0",
   "filename": "jquery.turbolinks.min.js",
   "author": "Sasha Koss <kossnocorp@gmail.com>",
```
Comment (extend1994) [inline ajax/libs/jquery.turbolinks/package.json]: Don't we need to keep `Rico Sta. Cruz <hi@ricostacruz.com>` in [package.json](https://github.com/kossnocorp/jquery.turbolinks/blob/master/package.json#L6)? He seems to contribute a lot in this library.

File: ajax/libs/jquery.turbolinks/package.json [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
   "description": "jQuery plugin for drop-in fix binded events problem caused by Turbolinks",
   "license": "MIT",
   "keywords": [
     "jquery.turbolinks"
```
Comment (extend1994) [inline ajax/libs/jquery.turbolinks/package.json]: Maybe more keywords?

**Comment by sufuf3** [comment]
@extend1994 I modified the authors and keywords
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@hare1039 @qwe19272375 @dakshshah96 
Please help me review this PR, Thank you.

**Pull request merged**